terraform/helm/genesis/values.yaml

```diff
@@ -5,7 +5,7 @@ chain:
   root_key: '0x5243ca72b0766d9e9cbf2debf6153443b01a1e0e6d086c7ea206eaf6f8043956'
   min_stake: 0
   max_stake: 100000
-  epoch_duration_secs: 86400 # 1 day
+  epoch_duration_secs: 120 # 2 minutes
```

Comment:
I am not sure if we want to do it by default for every forge run - how do we think about having a nightly suite that can set very aggressive epoch change and run it for a longer duration to catch some of the regressions around epoch changes?

Reply:
how do I change the config only for forge run for PRs? I think that is not possible yet, so using 2 minutes for all until then should be fine too?

I think we should have at least one or two epoch changes for push-blocking forge run (which lasts 5 minutes)?

Reply:
@igor-aptos - How about we hold off on this while we add support for adding node configuration support for Forge? I will take a stab at it and hopefully we have it by early next week?